tests/test_dashboard/test_views.py

```diff
@@ -765,6 +765,15 @@ def test_permission_classes(self):
         view_class = view_func.view_class
         self.assertEqual(view_class.permission_classes, [FXHasTenantCourseAccess])
 
+    def test_get_related_id(self):
+        """Verify get_related_id returns course_id"""
+        view_func, _, kwargs = resolve(self.url)
+        view = view_func.view_class()
+        view.kwargs = kwargs
+        expected_related_id = 'course-v1:ORG1+5+5'
+        related_id = view.get_related_id()
+        self.assertEqual(expected_related_id, related_id)
```

Comment:
Please prefer plain asserts for better debugging using `pytest` and slightly faster performance than using old methods.

```suggestion
        assert expected_related_id == related_id
```